common.mk

```diff
@@ -395,7 +369,9 @@ OPTEE_OS_COMMON_FLAGS ?= \
 optee-os-common:
 	$(MAKE) -C $(OPTEE_OS_PATH) $(OPTEE_OS_COMMON_FLAGS)
 
-OPTEE_OS_CLEAN_COMMON_FLAGS ?= $(OPTEE_OS_COMMON_EXTRA_FLAGS)
+OPTEE_OS_CLEAN_COMMON_FLAGS ?= \
```

Comment:
Why is this needed at all? Couldn't `OPTEE_OS_COMMON_FLAGS` be used instead?

Reply:
Agree. Could use `OPTEE_OS_COMMON_FLAGS`.

I don't known if anyone uses a local `OPTEE_OS_CLEAN_COMMON_FLAGS`. So I wonder if we should use
```makefile
OPTEE_OS_CLEAN_COMMON_FLAGS ?= $(OPTEE_OS_COMMON_FLAGS)
```
or use straight at line 380 below
```makefile
optee-os-clean-common: xtest-clean-common optee-examples-clean-common
	$(MAKE) -C $(OPTEE_OS_PATH) $(OPTEE_OS_COMMON_FLAGS) clean
```


Note: maybe some other simplification could be made on other `*_CLEAN_COMMON_FLAGS`.

Reply:
I'd prefer the latter and get rid of `OPTEE_OS_CLEAN_COMMON_FLAGS` completely.

> Note: maybe some other simplification could be made on other `*_CLEAN_COMMON_FLAGS`.

Not necessarily in scope here. Do as you like.

Reply:
thanks for the feedback